node.js/index.js

```diff
@@ -0,0 +1,114 @@
+// External Libraries:
+const agent = require('agentkeepalive');
+const request = require('request');
+
+// Internal Libraries:
+const constants = require('./constants');
+
+// Getting Environment Variables:
+const getConfig = () => {
+    var config = {};
+
+    if (process.env.LOGDNA_KEY) config.key = process.env.LOGDNA_KEY;
+    if (process.env.LOGDNA_HOSTNAME) config.hostname = process.env.LOGDNA_HOSTNAME;
+
+    if (process.env.LOGDNA_TAGS && process.env.LOGDNA_TAGS.length > 0) {
+        config.tags = process.env.LOGDNA_TAGS.split(',').map(tag => tag.trim()).join(',');
+    }
+
+    return config;
+};
+
+// Message Sanity Check:
+const sanitizeMessage = message => {
+    var sanitized = message.substring(0, constants.MAX_LINE_LENGTH);
+    if (sanitized !== message) sanitized += ' (cut off, too long...)';
+    return sanitized;
+};
+
+// Parsing the gzipped Log Data:
```

Comment:
`// Parse the gzipped log data`

Reply:
I capitalize almost all since it forces us to use only important words in the description.